examples/app-claim.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: platform.upbound.io/v1alpha1
+kind: App
+metadata:
+  name: platform-ref-gcp-ghost
+  namespace: default
+  labels:
+    platform.upbound.io/deletion-ordering: enabled
+spec:
+  compositeDeletePolicy: Foreground
+  parameters:
+    providerConfigName: platform-ref-gcp
+    passwordSecretRef:
+      namespace: default
+      name: platform-ref-gcp-db-conn-mariadb
```

Comment:
it's not working until we add mariadb support to https://github.com/upbound/configuration-gcp-database, correct? Just to double check that it is a placeholder

Reply:
Yes